ui.frontend/src/view/FormFieldBase.js

```diff
@@ -130,6 +175,34 @@ export default class FormFieldBase extends FormField {
         this.widget.value = value;
     }
 
+    /**
+     * Shows or Hides Description Based on click of '?' mark.
+     * @private
+     */
+    _addHelpIconHandler(state) {
+        const questionMarkDiv = this.getQuestionMarkDiv(),
+            descriptionDiv = this.getDescription(),
+            tooltipAlwaysVisible = this._isTooltipAlwaysVisible(state);
+        const self = this;
+        if (questionMarkDiv && descriptionDiv) {
+            questionMarkDiv.onclick = function(e) {
+                e.preventDefault();
```

Comment:
why do we need this ?

Reply:
the best way is to add an event listener
```
.addEventListener('click', () => {

})
```

Reply:
> why do we need this ?

on click of ? icon, we need to show the description field